recipe/meta.yaml

```diff
@@ -1,128 +1,82 @@
 {% set name = "sktime" %}
-{% set version = "0.17.0" %}
+{% set version = "0.24.1" %}
 
 package:
   name: sktime-suite
   version: {{ version }}
 
 source:
-  url: https://github.com/sktime/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 75cb18dbc2fc4e15316130e1401e676f568faade8b711b5140e8c2fd1ff60b06
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9d5ac84546676fd64fb05bb5fc077d38703bea3bcf711885e992da7bf12f6b83
 
 build:
   number: 0
-  # numba is not available on s390x
-  skip: true  # [s390x]
-  skip: true  # [py<37 or py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
-# This section is required to correctly rerender skip statement
 requirements:
   host:
     - python
     - pip
     - setuptools
     - wheel
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - packaging
+    - pandas >=1.1,<2.2.0
+    - skbase <0.7.0
+    - scikit-learn >=0.24,<1.4.0
+    - scipy >=1.2.0,<2.0.0
+  run_constrained:
```

Comment:
run_constrained's purpose is to add compatibility constraints. 
All dependencies without constraints should be removed (e.g. cloudpickle, dask).

I also see some mismatch with https://github.com/sktime/sktime/blob/v0.24.1/pyproject.toml#L79

Reply:
thanks for the tip!